apps/dog-food/Main.hs

```diff
@@ -70,19 +70,31 @@ data ChatCommand
   | Connect Contact SMPQueueInfo
   | DeleteConnection Contact
   | SendMessage Contact ByteString
+  | NewGroup Group
+  | AddToGroup Group Contact
+  | RemoveFromGroup Group Contact
+  | DeleteGroup Group
+  | ListGroup Group
+  | SendGroupMessage Group ByteString
+  deriving (Show)
 
 chatCommandP :: Parser ChatCommand
 chatCommandP =
   ("/help" <|> "/h") $> ChatHelp
-    <|> ("/markdown" <|> "/m") $> MarkdownHelp
+    <|> ("/group #" <|> "/g #") *> (NewGroup <$> group)
```

Comment:
add these to help? not a long explanation of workflow as for connections, but a list of commands and what they do

Reply:
even if we don't want to announce this as a feature for public use, make a release and explain in readme, at least help would be helpful for ourselves

Reply:
I'd rather not... readme is related to the current release at the moment, we can merge it to master but not advertise.